### ISSUE TITLE
clickin outside of color picker doesn't close the color window on ios

### ISSUE THREAD

**Issue by shivenigma** [ruhley/angular-color-picker#82]
I am using this inside an hybrid app using ionic framework, this works fine with android devices. But in iOS the color picker is not closed when I am clicking outside of the color picker.

I'm using like this 
```
<color-picker
                       class="color-picker-dir"
                       ng-model="vm.textColor"
                       color-picker-format="'hex'"
                       color-picker-pos="'top left'"
                       color-picker-swatch="true"
                       color-picker-swatch-only="true"
                       color-picker-case="'lower'"
                       color-picker-inline="false"
               ></color-picker>
```

**Comment by ruhley** [comment]
I don't have an iOS device handy, so I will try to get to this when I can get a hold of one.

**Comment by shivenigma** [comment]
Thanks, I modified the directive to have a little done button on top of the directive. That'll do for my requirment. But I have another question, Can we implement touch support like dragging the color selector by touching and dragging? I think that may not be hard to implement but it'll have negative impacts on performance.

**Comment by ruhley** [comment]
Fixed and released in v2.2.0
